tests/functional/storageclass/test_rbd_default_storageclass.py

```diff
@@ -52,12 +47,11 @@ def test_pvc_creation_without_storageclass_name(self, pvc_factory, pod_factory):
         assert pvc_obj, "PVC creation failed."
 
         sc_attached_to_pvc = pvc_obj.get().get("spec").get("storageClassName")
-        sc_default_in_cluster = default_storage_class(constants.CEPHBLOCKPOOL)
         log.info("Verifying the storageclass attached to PVC is correct.")
 
-        assert (
-            sc_attached_to_pvc == sc_default_in_cluster.name
-        ), "Storageclass attached to PVC is different from StorageClass set as default for BlockPool."
+        assert is_rbd_default_storage_class(
+            sc_name=sc_attached_to_pvc
+        ), "RBD is not default storageclass for Cluster."
```

Comment:
If we have multiple RBD storageclasses and one is default, will this message be good enough even then ?

Reply:
Here, we are specifically checking the 'default' tag  on the storageclass which is attached to the PVC.  For more clarity, I will add the storage class name in the assert message.